tests/finance/test_transaction.py

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright 2017 Quantopian, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import pandas as pd
+from unittest import TestCase
+
+from zipline.assets import Asset
+from zipline.finance.transaction import Transaction
+
+
+class TransactionTestCase(TestCase):
+
+    def test_transaction_repr(self):
+        dt = pd.Timestamp('2017-01-01')
+
+        asset = Asset(1, exchange='test')
```

Comment:
I would probably make this an `Equity` or a `Future`.  The base `Asset` class isn't really meant to be instantiated directly.